README.md

```diff
@@ -1,4 +1,4 @@
 # tesp [![Build Status](https://travis-ci.org/OpenDataCubePipelines/tesp.svg?branch=master)](https://travis-ci.org/OpenDataCubePipelines/tesp)
 
-OpenDataCubePipeline code for the processing of Australasian Regional Copernicus Data Hub L1C S2A/B data
+OpenDataCubePipeline is code for processing Australasian Regional Copernicus Data Hub L1C S2A/B data.
```

Comment:
Probably worth a proper revisit on this one.  It's the pipeline code for the DEA's Optical ARD.

* Landsat 5
* Landsat 7
* Landsat 8
* Sentinel 2A
* Sentinel 2B

Reply:
Cool, I've updated the README.  If you're ok with it now feel free to hit the merge and blow this branch away <Capow>